### DISCUSSION TITLE
智慧運動訓練-棒球！⚾

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#117] [General]
**新聞內容:**
由「Keep Tossing Lab 傳接球實驗室」所研發的「**TrendUp.me™全方位棒球數據表現追蹤**」技術，可透過高清攝影鏡頭提供打擊慢動作錄影、以統計學為基礎的資料分析，提供打擊與投球的數據視覺化及雲端資料庫的服務，讓球員、教練以及訓練師都能隨時隨地追蹤運動表現。
由於職業選手打擊速度動輒超過 140 公里，因此打擊分面也整合「**GARMIN IMPACT 揮棒訓練分析儀**」，透過在球棒末端分析儀，即可取得當次的揮棒指標提供客觀數據協助教練以此為調整基礎，也能在手機或電腦上查看分析數據與 3D 揮棒路徑。
另外，貝恩沛拉棒球學校將棒球與 VR 兩相結合，推出「**VR 沉浸式體感訓練系統**」。此套系統具備 VR 體感、數據分析以及智慧學習等服務，僅需 4.5mx4.5m 的室內空間，就能模擬出不同型態的球場。
投球部分也採用國立清華大學運動科技中心籌組棒球科技運動團隊開發的「**STRIKE 智慧棒球 2.0**」進行投球訓練。該智慧棒球內置入多項精密的物聯網多軸感測器，在投擲過程中，可自動偵測轉速、球速、出手時間、投球時間等關鍵數值，以及球棒運動軌跡、回饋力道與揮擊點位，其用料或是材質皆與競賽用球相同。

**新聞心得:**
上面四項針對運動訓練的分析我覺得對於選手平時訓練很有幫助，尤其是打擊跟投球的部分，可以測到以人力無法去計算的數值，並且累積數據分析，讓選手跟教練運用來加強訓練，而VR 體感訓練可以以小小的空間模擬實際場地的感覺真的很酷，或許以後很多運動都可以發展出相關的高科技輔助訓練?!

**新聞來源:**
https://news.cnyes.com/news/id/4770798?exp=a
![7dd7708f10052a67f89613fcb7d1e3b7](https://user-images.githubusercontent.com/90883526/141887166-cca87795-1411-44cf-87f7-744e2c6e3f3f.jpg)

**Comment by pocper1** [comment]
打棒球都需要戶外場地，如果再室內也可以練習就不用看天氣來練球了，對於選手是一大福音。

**Comment by briankuo123** [comment]
現在甚麼都可以套用AI，我是不是夠去期待有了AI的訓練，以後的棒球比賽能夠越來越精彩。

**Comment by minniewei** [comment]
從這次專題可以知道要做到這麼精細的模型製作一定要花很多的功夫蒐集很多資料

**Comment by wh20502** [comment]
以前的人戰勝140以上球速的人稱神，以後可能訓練完這些都是基本款了。

**Comment by Shiba010** [comment]
太酷了！透過ai的訓練，以後的運動賽事一定更加精彩可期！

**Comment by Yunhans** [comment]
我記得之前也有看過網球的

**Comment by LanningYang** [comment]
個人是看過關於賽跑利用AI，來調整選手的動作
